pkg/types/types.proto

```diff
@@ -100,11 +100,12 @@ message WorkflowInvocationStatus {
     google.protobuf.Timestamp updatedAt = 2;
     map<string, TaskInvocation> tasks = 3;
     TypedValue output = 4;
+    TypedValue outputHeaders = 5;
 
     // In case the task ID also exists in the workflow spec, the dynamic task will be
     // used as an overlay over the static task.
-    map<string, Task> dynamicTasks = 5;
-    Error error = 6; // Only set when status == failed
+    map<string, Task> dynamicTasks = 6;
+    Error error = 7; // Only set when status == failed
```

Comment:
Don't change the numbers of existing fields. This will break compatibility with earlier versions. Just name outputHeaders = 7 ;)